### PR TITLE
[Form] fixed BC-break on grouped choice lists

### PR DESCRIPTION
**Pull request by origaminal**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15037 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by stof** [comment]
@webmozart ping

**Comment by stof** [comment]
:+1: 

Status: reviewed

File: src/Symfony/Component/Form/Tests/ChoiceList/Factory/DefaultChoiceListFactoryTest.php [hunk 1]
```diff
@@ -758,6 +758,36 @@ public function testCreateViewForLegacyChoiceList()
         $this->assertEquals(array(new ChoiceView('x', 'x', 'Preferred')), $view->preferredChoices);
     }
 
+    /**
+     * @group legacy
+     */
+    public function testCreateViewForLegacyChoiceList()
```
Comment (webmozart) [inline src/Symfony/Component/Form/Tests/ChoiceList/Factory/DefaultChoiceListFactoryTest.php]: I'd name this `testCreateViewForNestedLegacyChoiceList` for clarity.

**Comment by webmozart** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @origaminal.

**Comment by trsteel88** [comment]
Just wondering what the status of this patch is? I can't see it merged in anywhere and am still having the same issue on the latest 2.7.

Thanks

**Comment by xabbuh** [comment]
@trsteel88 The commits have been squashed before applying them to the `2.7` branch. That's why the PR is shown as closed. The changes will be part of Symfony 2.7.4 (the tag is already available).

**Comment by trsteel88** [comment]
Thanks @xabbuh

**Pull request closed without merging**